src/gf/meta.py

```diff
@@ -92,6 +92,24 @@ class Result(SeismosizerResult):
 class StaticResult(SeismosizerResult):
     result = Dict.T()
 
+    def return_kite_scene(self, satellite_target, scene_config, component,
```

Comment:
Let's simply call it `kite_scene(...)`. This would be in line with `pyrocko_trace()`, etc. of other result objects.

Reply:
Common format is good. I would also like to have the kite_scene() inputs minimized to self as for the pyrocko_traces(), however i do not see how to reduce the input needed and still get a fully informed kite scene (@miili ?)

Reply:
I added a SeismosizerSatelliteScene similar to SeismosizerTrace that get returned alongside the static result if it is of instance SatelliteTarget. That way it is similar to the waveforms and can carry any information down, e.g. here to kite_scene(), which only needs now the desired component.  SatelliteTarget now also takes the optional nrows and ncols inputs. Changes are all optional and should not have an effect on any previous instances of handling a SatelliteTarget.